static/js/components/dashboard/util.js

```diff
@@ -16,3 +17,26 @@ export const courseListToolTip = (text: string, id: string) => (
     </ReactTooltip>
   </div>
 );
+
+export const isCurrentlyEnrollable = (enrollmentStartDate, now) => (
```

Comment:
flow was choking on some issue yesterday, and alice nor i could figure it out, but i read something last night that gave me a hint. i'll look into this again